.gitignore

```diff
@@ -86,6 +86,7 @@ ipython_config.py
 #   For a library or package, you might want to ignore these files since the code is
 #   intended to run in multiple environments; otherwise, check them in:
 # .python-version
+env
```

Comment:
if you check, line 122...all the possible environment files are in the .gitignore file.

Reply:
Ok thank my bad on that